packages/api/src/config/config.module.ts

```diff
@@ -0,0 +1,21 @@
+import { DynamicModule, Global, Module } from "@nestjs/common";
+
+import { BrevoModuleConfig } from "./brevo-module.config";
+import { BREVO_MODULE_CONFIG } from "./brevo-module.constants";
+
+@Global()
+@Module({})
+export class ConfigModule {
+    static forRoot(config: BrevoModuleConfig): DynamicModule {
```

Comment:
IMO there's no need for this to be a separate module. The Brevo module should provide the config

Reply:
I wanted to extract this into a seperate config module so it can easily imported in further modules as well (target groups, campaigns,..). Else the config must be manually passed to each module in the (root) BrevoModule and. I think this keeps the modules a bit cleaner. Do you still think it would be better to not have the separate module?